### PR TITLE
Log Cleanup for Dataset Factory

### PR DESCRIPTION
**Pull request by deepakwadhwani**
Fixes for : 
https://issues.cask.co/browse/CDAP-11534
https://issues.cask.co/browse/CDAP-11460

### CODE REVIEW

File: cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/deploy/pipeline/StreamCreator.java [hunk 1]
```diff
@@ -55,7 +59,9 @@ void createStreams(NamespaceId namespaceId, Iterable<StreamSpecification> stream
       if (ownerPrincipal != null) {
         props.put(Constants.Security.PRINCIPAL, ownerPrincipal.getPrincipal());
       }
-      streamAdmin.create(namespaceId.stream(spec.getName()), props);
+      if (streamAdmin.create(namespaceId.stream(spec.getName()), props) != null) {
+        LOG.info("Stream '{}' created successfully.", spec.getName(), props);
```
Comment (anew) [inline cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/deploy/pipeline/StreamCreator.java]: should the log message have the namespace id?

File: cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/DefaultSchedulerService.java [hunk 2]
```diff
@@ -54,8 +54,8 @@
 
     @Override
     public void execute(JobExecutionContext context) throws JobExecutionException {
-      LOG.debug("Trying to run job {} with trigger {}", context.getJobDetail().getKey().toString(),
-                context.getTrigger().getKey().toString());
+      LOG.debug("Emitting time notification for program '{}' and schedule '{}'.",
+                context.getJobDetail().getKey().toString(), context.getTrigger().getKey().toString());
       Trigger trigger = context.getTrigger();
```
Comment (anew) [inline cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/DefaultSchedulerService.java:59]: unrelated change? or is this fixing an incorrect message?

**Review by anew — COMMENTED** [review]
two minor comments.

File: cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/DefaultSchedulerService.java (hunk 2 above)
Comment (deepakwadhwani) [reply cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/DefaultSchedulerService.java:59]: @anew this change is regarding your comment in the bug https://issues.cask.co/browse/CDAP-11534.

Comment (anew) [reply cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/DefaultSchedulerService.java:59]: ok

**Review by anew — APPROVED** [review]
LGTM

**Comment by deepakwadhwani** [comment]
Clean build: https://builds.cask.co/browse/CDAP-RUT1004-1
